generate.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const generate: (charset: string, length: number) => string;
```

Comment:
the actual [`generate`](https://github.com/ai/nanoid/blob/ef1d664b52fb0cd978371c9601658f09c84c88cc/generate.js#L21) functions's arguments are `alphabet` and `size`.